.github/workflows/release.yml

```diff
@@ -22,14 +22,17 @@ jobs:
       - name: Install dependencies and build
         env:
           DEFI_SDK_API_URL: wss://api-v4.zerion.io/
+          DEFI_SDK_API_URL: wss://api-testnet.zerion.io/
           ZERION_API_URL: https://zpi.zerion.io/
+          ZERION_API_URL: https://zpi-testnet.zerion.io/
           PROXY_URL: https://proxy.zerion.io/
           DEFI_SDK_TRANSACTIONS_API_URL: https://transactions.zerion.io
           DEFI_SDK_API_TOKEN: Zerion.0JOY6zZTTw6yl5Cvz9sdmXc7d5AhzVMG
           SOCIAL_API_URL: https://social.zerion.io/
           TEST_WALLET_ADDRESS: ${{ secrets.TEST_WALLET_ADDRESS }}
           FEATURE_SEND_FORM: on
           FEATURE_FOOTER_BUG_BUTTON: off
+          FEATURE_PAYMASTER: off
```

Comment:
hm, should this feature be 'on' for testnet release?

Reply:
I don't know yet, we'll see :)